pkg/apis/pipeline/v1/openapi_generated.go

```diff
@@ -30,68 +30,68 @@ import (
 
 func GetOpenAPIDefinitions(ref common.ReferenceCallback) map[string]common.OpenAPIDefinition {
 	return map[string]common.OpenAPIDefinition{
-		"github.com/tektoncd/pipeline/pkg/apis/pipeline/pod.AffinityAssistantTemplate":   schema_pkg_apis_pipeline_pod_AffinityAssistantTemplate(ref),
```

Comment:
also I am confused by how `./hack.update-codegen` has behaved on my local env of m1 macOS, I have put the tekton/pipeline repo under $GOPATH but this seems to be very wrong of changing schema_pkg_apis ref to local 🤔

Reply:
I'm not sure what the "recommended" approach is but I don't set GOPATH, and the scripts should set up a temporary GOPATH for codegen. Try clearing that env var, running codegen again, and if it's still not working for you, you can open an issue.

Reply:
Thanks, I have tried unsetting the env variable. But it sill swapped all github paths to my local paths and caused the diff issue. I opened issue to track at https://github.com/tektoncd/pipeline/issues/6485